docs/reference/ccr/index.asciidoc

```diff
@@ -10,6 +10,7 @@ cases:
 * Disaster recovery in case a primary cluster fails. A secondary cluster can
   serve as a hot backup
 * Geo-proximity so that reads can be served locally
+* Optimizing indexing throughput by offloading search to a secondary cluster
```

Comment:
The data needs to be replicated to the follower cluster as well, so I'm not sure if the phrasing here (optimize indexing throughput) makes it clear what the actual benefit is (namely to better handle (search) load spikes, so that they don't interfere with indexing).